### PR TITLE
Add changes to CHANGELOG.md for 2.4.1

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,9 +32,14 @@
 
 * Deere: make sampler rows persist [#12928](https://github.com/mixxxdj/mixxx/pull/12928)
 * Tango: Remove unneeded waveform Singleton [#12938](https://github.com/mixxxdj/mixxx/pull/12938)
-* Possible crash in customs skins using parallel waveforms [#13043](https://github.com/mixxxdj/mixxx/pull/13043) [#12580](https://github.com/mixxxdj/mixxx/issues/12580)
+* Possible crash in customs skins using parallel waveforms
```
Comment (ronso0) [inline CHANGELOG.md:35]:
```suggestion
* Prevent possible crash in customs skins using parallel waveforms
```
or did we introduce a crash easter eg : )

File: CHANGELOG.md [hunk 2]
```diff
@@ -66,7 +84,18 @@
 * AutoDJ: Fix button state after error message about playing deck 3/4
   [#12976](https://github.com/mixxxdj/mixxx/pull/12976)
   [#12975](https://github.com/mixxxdj/mixxx/issues/12975)
-* Tagfetcher: Cache fetched covers [#12301](https://github.com/mixxxdj/mixxx/pull/12301) [#11084](https://github.com/mixxxdj/mixxx/issues/11084)
+* Tagfetcher: Cache fetched covers
+  [#12301](https://github.com/mixxxdj/mixxx/pull/12301)
+  [#11084](https://github.com/mixxxdj/mixxx/issues/11084)
+* Avoid iterator being one off and DEBUG_ASSERT in Beats::iteratorFrom
```
Comment (ronso0) [inline CHANGELOG.md:90]:
```suggestion
* Avoid beats iterator being one off and DEBUG_ASSERT in Beats::iteratorFrom
```

**Comment by daschuer** [comment]
Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -66,7 +84,18 @@
...
-* Tagfetcher: Cache fetched covers [#12301](https://github.com/mixxxdj/mixxx/pull/12301) [#11084](https://github.com/mixxxdj/mixxx/issues/11084)
+* Tagfetcher: Cache fetched covers
+  [#12301](https://github.com/mixxxdj/mixxx/pull/12301)
+  [#11084](https://github.com/mixxxdj/mixxx/issues/11084)
+* Avoid beats iterator being one off and DEBUG_ASSERT in Beats::iteratorFrom
+  [#13150](https://github.com/mixxxdj/mixxx/pull/13150)
+  [#13149](https://github.com/mixxxdj/mixxx/issues/13149)
+* Show hint if resource path in CMakeCache.txt does not exist
+  [#12929](https://github.com/mixxxdj/mixxx/pull/12929)
+* Always calculate the auto value for colorful console output. [#13153](https://github.com/mixxxdj/mixxx/pull/13153)
```
Comment (ronso0) [inline CHANGELOG.md:95]:
```suggestion
* Always calculate the auto value for colorful console output [#13153](https://github.com/mixxxdj/mixxx/pull/13153)
```

Comment (ronso0) [reply CHANGELOG.md:95]: Just this .
squash & push and please feel free to merge yourself!

**Review by ronso0 — APPROVED** [review]

**Review by Swiftb0y — APPROVED** [review]
LGTM. waiting for the remaining comment from ronso0 to get fixed

**Comment by ronso0** [comment]
Maybe we can agree to use "Maintainers are allowed to edit this pull request." for such minor, final fixups?
Then, optionally, `gh pr checkout -f ...`, sqash (or reset HEAD^1 and amend), push, merge, done.

Wdyt?

**Comment by daschuer** [comment]
The flag is already set: 
grafik

Work is in progress.

**Comment by ronso0** [comment]
> The flag is already set:

yeah, I know, it's just we usually don't make use of it without asking first.

**Comment by ronso0** [comment]
Thanks!

**Pull request merged**